### PR TITLE
release: refine 5.3.3 release note

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
release: refine 5.3.3 release note

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by heipark — COMMENTED** [review]
LGTM

File: releases/release-5.4.2.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ title: TiDB 5.4.2 Release Notes
 
 TiDB 版本：5.4.2
 
+> **警告：**
+>
+> 不建议升级至 v5.4.2 版本，因为该版本已知存在 bug，详情参见 [#12934](https://github.com/tikv/tikv/issues/12934)。该 bug 会在未来的 v5.4 版本中修复。
```
Comment (heipark) [inline releases/release-5.4.2.md]: 该 bug 会在 v5.4.3 修复 ??

Comment (TomShawn) [inline releases/release-5.4.2.md:13]:
```suggestion
> 不建议升级至 v5.4.2 版本，因为该版本已知存在 bug，详情参见 [#12934](https://github.com/tikv/tikv/issues/12934)。该 bug 会在未来的 v5.4.3 版本中修复。
```

**Review by heipark — APPROVED** [review]

File: releases/release-5.3.2.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ title: TiDB 5.3.2 Release Notes
 
 TiDB 版本：5.3.2
 
+> **警告：**
+>
+> 不建议使用 v5.3.2 版本，因为该版本已知存在 bug，详情参见 [#12934](https://github.com/tikv/tikv/issues/12934)。该 bug 已在 v5.3.3 中修复，建议升级至 [v5.3.3 版本](/releases/release-5.3.3.md)。
```
Comment (TomShawn) [inline releases/release-5.3.2.md:13]:
```suggestion
> 不建议使用 v5.3.2，因为该版本已知存在 bug，详情参见 [#12934](https://github.com/tikv/tikv/issues/12934)。该 bug 已在 v5.3.3 中修复，建议升级至 [v5.3.3 版本](/releases/release-5.3.3.md)。
```

File: releases/release-5.4.2.md [hunk 3]
```diff
@@ -8,6 +8,10 @@ title: TiDB 5.4.2 Release Notes
 
 TiDB 版本：5.4.2
 
+> **警告：**
+>
+> 不建议使用 v5.4.2 版本，因为该版本已知存在 bug，详情参见 [#12934](https://github.com/tikv/tikv/issues/12934)。该 bug 会在未来的 v5.4.3 版本中修复。
```
Comment (TomShawn) [inline releases/release-5.4.2.md:13]:
```suggestion
> 不建议使用 v5.4.2，因为该版本已知存在 bug，详情参见 [#12934](https://github.com/tikv/tikv/issues/12934)。该 bug 会在未来的 v5.4.3 版本中修复。
```

File: releases/release-5.4.2.md [hunk 4]
```diff
@@ -8,6 +8,10 @@ title: TiDB 5.4.2 Release Notes
 
 TiDB 版本：5.4.2
 
+> **警告：**
+>
+> 不建议使用 v5.4.2，因为该版本已知存在 bug，详情参见 [#12934](https://github.com/tikv/tikv/issues/12934)。该 bug 会在未来的 v5.4.3 版本中修复。
```
Comment (Oreoxmt) [inline releases/release-5.4.2.md:13]:
```suggestion
> 不建议使用 v5.4.2，因为该版本已知存在 bug，详情参见 [#12934](https://github.com/tikv/tikv/issues/12934)。该 bug 会在未来的 v5.4.3 中修复。
```

File: releases/release-5.3.2.md [hunk 5]
```diff
@@ -8,6 +8,10 @@ title: TiDB 5.3.2 Release Notes
 
 TiDB 版本：5.3.2
 
+> **警告：**
+>
+> 不建议使用 v5.3.2，因为该版本已知存在 bug，详情参见 [#12934](https://github.com/tikv/tikv/issues/12934)。该 bug 已在 v5.3.3 中修复，建议升级至 [v5.3.3 版本](/releases/release-5.3.3.md)。
```
Comment (Oreoxmt) [inline releases/release-5.3.2.md:13]:
```suggestion
> 不建议使用 v5.3.2，因为该版本已知存在 bug，详情参见 [#12934](https://github.com/tikv/tikv/issues/12934)。该 bug 已在 v5.3.3 中修复，建议升级至 [v5.3.3](/releases/release-5.3.3.md)。
```

File: releases/release-5.3.3.md [hunk 6]
```diff
@@ -14,10 +14,8 @@ TiDB 版本：5.3.3
 
     - 修复了 PD leader 发生切换或重启 PD 后，在集群中执行 SQL 语句会出现持续报错的问题。
 
-        该问题是由于 TiKV 存在 bug，TiKV 向 PD client 发送心跳请求失败后不会重试，只能等待与 PD client 重连。这样，故障 TiKV 节点上的 Region 的信息会逐步变旧，使得 TiDB 无法获取最新的 Region 信息，导致 SQL 执行出错。
-
-        目前该问题已在 v5.3 版本上修复。你可以升级集群至 v5.3.3。
-
-        你还可以重启无法向 PD 发送 Region 心跳的 TiKV 节点，直至不再有待发送的 Region 心跳为止。
+        - 问题原因：该问题是由于 TiKV 存在 bug，TiKV 向 PD client 发送心跳请求失败后不会重试，只能等待与 PD client 重连。这样，故障 TiKV 节点上的 Region 的信息会逐步变旧，使得 TiDB 无法获取最新的 Region 信息，导致 SQL 执行出错。
+        - 影响版本：v5.3.2 和 v5.4.2。目前该问题已在 v5.3.3 版本上修复。如果你使用 v5.3.2 版本的 TiDB 集群，可以升级至 v5.3.3。
```
Comment (Oreoxmt) [inline releases/release-5.3.3.md:18]:
```suggestion
        - 影响版本：v5.3.2 和 v5.4.2。目前该问题已在 v5.3.3 上修复。如果你使用 v5.3.2 的 TiDB 集群，可以升级至 v5.3.3。
```
要不要加一个 TiDB 集群升级的文档链接？

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**